src/views/virtualmachinesinstance/list/utils/virtualMachinesInstancesRowFilter.ts

```diff
@@ -0,0 +1,51 @@
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { FilterValue, RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+import { VM_KUBEVIRT_OS_ANNOTATION } from './virtualMachinesInstancesConstants';
+import { osNames, vmiStatuses } from './virtualMachinesInstancesStatuses';
+
+const osTitles = {
+  centos: 'Centos',
+  fedora: 'Fedora',
+  windows: 'Windows',
+  opensuse: 'OpenSuse',
+  rhel: 'RHEL',
+  ubuntu: 'Ubuntu',
+};
+
+const getOSName = (obj: V1VirtualMachineInstance): string | undefined => {
+  const osAnnotation = obj?.metadata?.annotations[VM_KUBEVIRT_OS_ANNOTATION];
+  const [osName] = osAnnotation?.split(/(\d.*)/);
+  return osName;
```

Comment:
if osName isn't included in `osTitles` return `Other` here.

Reply:
done